fixbackend/subscription/aws_marketplace.py

```diff
@@ -209,12 +209,12 @@ async def create_billing_entry(
                 usage = int(len(summaries) * month_factor)
                 on_payment_free_tier = product_tier.paid is False
                 if on_payment_free_tier or usage == 0:
-                    log.info(f"AWS Marketplace: customer {customer} has no usage")
+                    log.info(f"{kind}: subscription {subscription.id} has no usage")
```

Comment:
Let's add an easy to search prefix here: I often query `AWS Marketplace` in loki to investigate problems.